subprojects/common/time/src/main/kotlin/com/avito/time/TimeProvider.kt

```diff
@@ -1,12 +1,17 @@
 package com.avito.time
 
+import androidx.annotation.RequiresApi
 import java.io.Serializable
+import java.time.Duration
 import java.util.Date
 
 public interface TimeProvider : Serializable {
```

Comment:
Out of scope, but this interface becomes more and more shallow.
Different methods do primitive conversions and don't hide the real logic.
I see it as only one method `now(): Duration` and clients convert it to what they need (ms, seconds, Date, whatever).
I hope desugaring will help to do so.

Reply:
I agree